src/components/common/NavigationBar.js

```diff
@@ -1,340 +1,214 @@
+  SettingsIcon
+} from "@chakra-ui/icons";
 
-    }
+import {
+  HOME_PAGE,
+  LOGIN_PAGE,
+  EVENTS_PAGE,
+  CREATE_EVENT_PAGE,
+  MY_EVENTS_PAGE,
+  CREATE_FORUM_PAGE,
+  FORUMS_PAGE,
+  MY_FORUMS_PAGE,
+  MY_PROFILE_PAGE,
+  USER_SETTINGS_PAGE,
+  CREATE_RESOURCE_PAGE,
+  RESOURCES_PAGE,
+  NETWORK_PAGE,
+} from "../constants/Routes";
+
+import "../../styles/navbar-style.css";
+
+import GOOSE_IMAGE from "../../images/Goose.png"
 
-    async function handleCreateResources() {
+export default function NavigationBar() {
 
-        history.push(CREATE_RESOURCE_PAGE)
+  const history = useHistory();
 
+  function redirectToPage(page) {
+    return async function () {
+      history.push(page);
     }
-
-    async function handleResources() {
-
-        history.push(RESOURCES_PAGE)
+  }
+
+  const handleHome = redirectToPage(HOME_PAGE);
+  const handleSettings = redirectToPage(USER_SETTINGS_PAGE);
+  const handleEvents = redirectToPage(EVENTS_PAGE);
+  const handleMyEvents = redirectToPage(MY_EVENTS_PAGE);
+  const handleCreateEvent = redirectToPage(CREATE_EVENT_PAGE);
+  const handleMyProfile = redirectToPage(MY_PROFILE_PAGE);
+  const handleCreateForum = redirectToPage(CREATE_FORUM_PAGE);
+  const handleForums = redirectToPage(FORUMS_PAGE);
+  const handleMyForums = redirectToPage(MY_FORUMS_PAGE);
+  const handleCreateResources = redirectToPage(CREATE_RESOURCE_PAGE);
+  const handleResources = redirectToPage(RESOURCES_PAGE);
+  const handleNetwork = redirectToPage(NETWORK_PAGE);
+
+  const [error, setError] = useState("")
+  const { currentUser, logout } = useAuth();
+
+  async function handleLogout() {
+    setError("")
+
+    try {
+      await logout()
+      history.push(LOGIN_PAGE)
+    } catch {
+      setError("Failed to log out")
     }
-
-    async function handleNetwork() {
-
-        history.push(NETWORK_PAGE)
-
-    }
-
-    return (
-        <>
-            <Paper>
-                <AppBar position="fixed">
-                    <Container maxWidth="xl">
-                        <Toolbar disableGutters>
-                            <Box sx={{ flexGrow: 1, display: { xs: 'none', md: 'flex' } }}>
-                                <Typography
-                                    variant="h3"
-                                    noWrap
-                                    component="a"
-                                    href="/"
-                                    sx={{
-                                        mr: 2,
-                                        display: { xs: 'none', md: 'flex' },
-                                        fontFamily: 'monospace',
-                                        fontWeight: 700,
-                                        letterSpacing: '.3rem',
-                                        color: 'inherit',
-                                        textDecoration: 'none'
-                                    }}
-                                >
-                                    HonkedIn
```

Comment:
As a UI aspect thats typical in a lot of cases, what do you think about linking the home page to the "HonkedIn" logo/word also. As in keep the home button, but also link home in the logo. I think LinkedIn does that